file-handling-example/src/download_file/handler.ts

```diff
@@ -0,0 +1,32 @@
+import { OperationHandlerSetup } from "@trayio/cdk-dsl/connector/operation/OperationHandlerSetup";
+import { FileHandlingExampleAuth } from "../FileHandlingExampleAuth";
+import { DownloadFileInput } from "./input";
+import { DownloadFileOutput } from "./output";
+import { OperationHandlerResult } from "@trayio/cdk-dsl/connector/operation/OperationHandler";
+
+export const downloadFileHandler = OperationHandlerSetup.configureHandler<
+	FileHandlingExampleAuth,
+	DownloadFileInput,
+	DownloadFileOutput
+	>((handler) =>
+		handler.usingHttp((http) =>
+			http
+				.get(
+					"https://content.dropboxapi.com/2/files/download"
```

Comment:
should we maybe suggest to create a server in postman to use as a "downloable" server?

Reply:
ok, all is dropbox. Let's leave as is. 👍